### ISSUE TITLE
feature: Get version.

### ISSUE THREAD

**Issue by Arano-kai** [colobot/colobot#999]
Sometimes it is good to be aware of version dependent "features" (:
I suppose to return `int[] ver = {<major>, <minor>, <patch>};` or so...
As an option: `vercheck(int[] Arg) => <true if Arg[0-3] >= ver[0-3]`

**Comment by krzys-h** [comment]
I don't think this would be really useful for anything, if the CBot functions change in a backward-incompatible way for whatever reason (and we really try to avoid that) you would still get compile errors. The same applies to checking if we are running a version that added a new function.

**Comment by hexagonrecursion** [comment]
> I don't think this would be really useful for anything, if the CBot functions change in a backward-incompatible way for whatever reason (and we really try to avoid that) you would still get compile errors. The same applies to checking if we are running a version that added a new function.

I can theoretically imagine a mod author wanting to support multiple mutually incompatible version of the game by passing different filenames to factory() and produce() depending on game version
